index.html

```diff
@@ -0,0 +1,33 @@
+        <p id="img-caption">This is an image of Dr Bashir</p>
+        </div>
+        <p id="tribute-info"> this is the paragraph which we need</p>
+        <a id="tribute-link" target="_blank"> This is a link</a>
```

Comment:
- Kindy put these under a section tag as they are sections in your code
-

Reply:
- [ ] You can also add why you request this change 🤗 section tags are semantic elements, you can add the importance of using semantic tags in your code so the student understands better 👍🏿